nsxt/resource_nsxt_vpc_service_profile.go

```diff
@@ -41,16 +41,6 @@ var vpcServiceProfileSchema = map[string]*metadata.ExtendedSchema{
 	"revision":     metadata.GetExtendedSchema(getRevisionSchema()),
 	"tag":          metadata.GetExtendedSchema(getTagsSchema()),
 	"context":      metadata.GetExtendedSchema(getContextSchema(true, false, false)),
-	"is_default": {
-		Schema: schema.Schema{
-			Type:     schema.TypeBool,
-			Optional: true,
-		},
-		Metadata: metadata.Metadata{
-			SchemaType:   "bool",
-			SdkFieldName: "IsDefault",
-		},
-	},
```

Comment:
I don't think this has any value as a computed attribute, is that correct?
That would be because for any user-created profile this is always going to be False.

Reply:
correct